GDataXML-HTML.podspec

```diff
@@ -1,123 +1,35 @@
-  #
-  #  CocoaPods is smart about how it includes source code. For source files
-  #  giving a folder will include any h, m, mm, c & cpp files. For header
-  #  files it will include any header in the folder.
-  #  Not including the public_header_files will make all headers public.
-  #
-
-  s.source_files  = "Pod/Classes"
-  #s.exclude_files = "GDataXML-HTML"
-
-  # s.public_header_files = "Classes/**/*.h"
-
-
-  # ――― Resources ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
-  #
-  #  A list of resources included with the Pod. These are copied into the
-  #  target bundle with a build phase script. Anything else will be cleaned.
-  #  You can preserve files from being cleaned, please don't preserve
-  #  non-essential files like tests, examples and documentation.
-  #
-
-  # s.resource  = "icon.png"
-  # s.resources = "Resources/*.png"
-
-  # s.preserve_paths = "FilesToSave", "MoreFilesToSave"
-
-
-  # ――― Project Linking ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
-  #
-  #  Link your library with frameworks, or libraries. Libraries do not include
-  #  the lib prefix of their name.
-  #
-
-  # s.framework  = "SomeFramework"
-  # s.frameworks = "SomeFramework", "AnotherFramework"
-
-  s.library   = "xml2"
-  # s.libraries = "iconv", "xml2"
-
-
-  # ――― Project Settings ――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
-  #
-  #  If your library depends on compiler flags you can set them in the xcconfig hash
-  #  where they will only apply to your library. If you depend on other Podspecs
-  #  you can include multiple dependencies to ensure it works.
-
-  s.requires_arc = true
-
-  s.xcconfig = { "HEADER_SEARCH_PATHS" => "$(SDKROOT)/usr/include/libxml2" }
-  # s.dependency "JSONKit", "~> 1.4"
+  s.name				= "GDataXML-HTML"
+  s.version				= "1.2.0"
+  s.summary				= "An XML/HTML Parser for iOS and OSX, based on Google's GDataXML."
+
+  s.description			= <<-DESC
+							GDataXML implements parts of NSXML so it's easy to parse XML or HTML with a DOM style API including
+							XPath support.
+							This fork of the original GDataXML adds support for the HTMLparser module of
+							libxml2 and allows you to deal with non validating XML in a simple way.
+							DESC
+
+  s.homepage			= "https://github.com/graetzer/GDataXML-HTML"
+  s.license				= "Apache License, Version 2.0"
+  s.author				= { "Simon Grätzer" => "simon@graetzer.org" }
+  s.source				= { :git => "https://github.com/graetzer/GDataXML-HTML.git", :tag => s.version.to_s }
+
+  s.source_files		= "Pod/Classes"
+  s.library				= "xml2"
+  s.requires_arc		= true
+  s.xcconfig			= { "HEADER_SEARCH_PATHS" => "$(SDKROOT)/usr/include/libxml2 $(PODS_ROOT)/GDataXML-HTML/libxml" }
+
+# Make it accessible within a framework
+  s.module_map			= 'libxml/libxml.modulemap'
+  s.prepare_command		= <<-CMD
+    cat > "libxml/libxml.modulemap" << MAP
+module libxml [system] {
+    header "/usr/include/libxml2/libxml/tree.h"
```

Comment:
This one doesn't work on my machine. You should use `$SDKROOT/usr/include/libxml2/libxml/tree.h`, like here https://github.com/neonichu/SQLite.swift/blob/master/SQLite.swift.podspec#L25-L33